projects/step-core/src/lib/modules/table/components/table/table.component.scss

```diff
@@ -1,46 +1,55 @@
+@use '../../../../../../styles/core-mixins' as core;
+
+@include core.table-space-between-columns;
+
 :host {
   position: relative;
   display: block;
-}
 
-table {
-  width: 100%;
-}
+  table {
+    width: 100%;
+    border-spacing: 10px 0 !important;
+  }
 
-tr.mat-row:hover {
-  background: var(--darken-background-10);
-}
+  ::ng-deep .mat-form-field-infix {
+    width: auto !important;
+  }
 
-th > mat-form-field {
-  width: 90%;
-}
+  ::ng-deep tr > td.mat-cell:not(.mat-column-bulkSelection):not(.mat-column-actions) {
+    padding: 0 8px;
+  }
 
-.spinner-container {
-  position: absolute;
-  z-index: 1;
-  width: 100px;
-  height: 100px;
-  left: calc(50% - 50px);
-  top: calc(50% - 50px);
-}
+  tr.mat-row:hover {
+    background: var(--darken-background-10);
```

Comment:
As css variables are no longer used in ng2+ part, this rule is redundant

Reply:
@dvladir Is there already a hover effect when we remove it?

Reply:
@neogucky As I see in UI there is no hover effect at all now. Is it a bug?

Reply:
Seems like a bug, probably because the variable doesn't exist anymore

Reply:
removed it, still no hover effect